README.md

```diff
@@ -6,247 +6,218 @@ Midtrans (formerly known as Veritrans Indonesia).
 [![Gem Version](https://badge.fury.io/rb/veritrans.svg)](http://badge.fury.io/rb/veritrans)
 [![Build Status](https://travis-ci.org/veritrans/veritrans-ruby.svg?branch=master)](https://travis-ci.org/veritrans/veritrans-ruby)
 
-To see it in action, we have made 3 examples:
+To see it in action, we have made example:
 
 1. Sinatra, which demonstrate in as succint code as possible. Please [have a look here](https://github.com/veritrans/veritrans-ruby/tree/master/example/sinatra)
-2. Simplepay, demonstrated how to integrate Midtrans with a simple, succint Rails project. [Have a look](https://github.com/veritrans/veritrans-ruby/tree/master/example/rails/simplepay).
-3. Cable, demonstrate a chat-commerce app where the payment is handled with Midtrans. [Have a look](https://github.com/veritrans/veritrans-ruby/tree/master/example/rails/cable).
 
-## How to use (Rails)
+## How to use
 
 ### Add gem veritrans to Gemfile
 
 ```ruby
 gem 'veritrans'
+```
+```ruby
+gem install veritrans
 ```
 
     bundle install
 
-### Generate veritrans.yml
+### General setting
+Create instance of Midtrans client
 
-    rails g veritrans:install
+```ruby
+require 'veritrans'
+
+mt_client = Midtrans.new(
+    server_key: "your server key",
+    client_key: "your client key",
+    api_host: "https://api.sandbox.midtrans.com", # default
+    http_options: { }, # optional
+    logger: Logger.new(STDOUT), # optional
+    file_logger: Logger.new(STDOUT), # optional
+  )
+
+mt_client.status("order-id-123456")
+```
 
-### Create simple payment form (optional)
+also can set one by one
```

Comment:
wording should be more descriptive:
Alternatively, you can also set config by declaring each one like below: